qml.qrc

```diff
@@ -63,6 +63,8 @@
         <file alias="eprosima_logo_horizontal.png">resources/images/eprosima_logo_horizontal.png</file>
         <file alias="eprosima_logo.ico">resources/images/eprosima_logo.ico</file>
         <file alias="fastdds-monitor-logo.png">resources/images/fastdds-monitor-logo.png</file>
+        <file alias="monitor_by_eprosima.svg">resources/images/monitor_by_eprosima.svg</file>
```

Comment:
What is the purpose of this file?

Reply:
That was another design for the background. I'd remove it as it is no longer needed